src/plugins/1/accessors/index.md

```diff
@@ -1,4 +1,4 @@
 ---
 layout: full.html.hbs
-title: Accessors
+title: "Plugin context: accessors"
```

Comment:
I disagree with that change: I feel that adding an `Essentials` and a `Plugin Context` separators would be cleaner.